packages/next-swc/crates/next-core/src/next_shared/transforms.rs

```diff
@@ -98,7 +98,7 @@ pub fn get_next_image_rule() -> ModuleRule {
             ModuleRuleCondition::ResourcePathEndsWith(".svg".to_string()),
         ]),
         vec![ModuleRuleEffect::ModuleType(ModuleType::Custom(
-            StructuredImageModuleTypeVc::new().into(),
+            StructuredImageModuleTypeVc::new(Value::new(BlurPlaceholderMode::DataUrl)).into(),
```

Comment:
Now it's possible to flip to the _next/image mode by changing this:

```suggestion
            StructuredImageModuleTypeVc::new(Value::new(BlurPlaceholderMode::NextImage)).into(),
```